gtsfm/averaging/translation/translation_averaging_base.py

```diff
@@ -53,9 +57,10 @@ def create_computation_graph(
             i2Ui1_graph: dictionary of relative unit translations as a delayed task.
             wRi_graph: list of global rotations wrapped up in Delayed.
             scale_factor: non-negative global scaling factor.
+            gt_poses_graph: List of ground truth poses (wTi) wrapped as Delayed for computing metrics.
 
         Returns:
-            Delayed: global unit translations wrapped using dask.delayed.
+            Global unit translations wrapped as Delayed.
+            A GtsfmMetricsGroup with translation averaging metrics wrapped as Delayed. 
         """
-
-        return dask.delayed(self.run)(num_images, i2Ui1_graph, wRi_graph, scale_factor)
+        return dask.delayed(self.run, nout=2)(num_images, i2Ui1_graph, wRi_graph, scale_factor, gt_wTi_graph)
```

Comment:
is `nout` required? or just for readability? hadn't seen that before

Reply:
it tells dask how many outputs are expected. Lets us do things like iterating or unpacking on dask objects. https://docs.dask.org/en/latest/delayed-api.html#dask.delayed.delayed

Reply:
so I can do `wti_graph, ta_metrics = self.trans_avg_module.create_computation_graph(num_images, pruned_i2Ui1_graph, wRi_graph, gt_wTi_graph=gt_poses_graph)` in MVO.

Reply:
very cool. nice find!